spec/requests/api/v1/attendances_spec.rb

```diff
@@ -4,24 +4,41 @@
 
 RSpec.describe 'Api::V1::Attendances', type: :request do
   let!(:logged_in_user) { create(:confirmed_user) }
-  let!(:business) { create(:business, user: logged_in_user) }
-  let!(:child) { create(:child, business: business) }
+  let!(:business)       { create(:business, user: logged_in_user) }
+  let!(:child)          { create(:child, business: business) }
+
+  let!(:week_mid_date)   { Time.new(2021, 9, 8).in_time_zone }  # Wed, 8th Sept, 2021
+  let!(:week_start_date) { week_mid_date - 3.days }             # Sun, 5th Sept, 2021
```

Comment:
Instead of adding days, can we use the `.at_beginning_of_week` and `.at_end_of_week helpers` just to stay consistent?

Reply:
Make sense